### PR TITLE
Crate Product table

### LINKED ISSUE #6: WOS-D-002 系統需儲存產品的相關資訊：產品 ID、名稱描述、其他屬性、產品圖片、 價格、庫存產品數量、標籤

### PR DESCRIPTION
**Pull request by wj461**

### CODE REVIEW

File: Product_table.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+create table Product (
+    Id INT primary key not null,
+    Store_id INT not null,
+    Name char(256) not null,
+    Describe char(256),
+    Category_id INT,
+    Picture BYTEA,
+    Price INT not null,
+    Stock INT not null
```
Comment (nonorze-VOXX) [inline Product_table.txt]: sql language should Upper

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]

File: Product_table.txt (hunk 1 above)
Comment (wj461) [reply Product_table.txt]: ok

File: Product_table.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CREATE TABEL Product (
```
Comment (huhuhu0420) [inline Product_table.txt]: TABEL

Comment (wj461) [reply Product_table.txt]: I fix

File: Product_table.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE Product (
+    Id INT primary key not null,
+    Store_id INT not null,
+    Name char(256) not null,
+    Describe char(256),
+    Category_id INT,
+    Picture BYTEA,
+    Price INT not null,
+    Stock INT not null
```
Comment (nonorze-VOXX) [inline Product_table.txt]: do you think INT is sql language
but PRIMARY KEY,  NOT NULL and CHAR  is not  sql language?

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]

File: Product_table.txt (hunk 3 above)
Comment (wj461) [reply Product_table.txt]: you are right
I fix

**Review by nonorze-VOXX — APPROVED** [review]
look good for now
someone say describe can use TEXT

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]
new request: put in db folder and sub filename request .sql

**Review by nonorze-VOXX — APPROVED** [review]

**Pull request merged**